### PR TITLE
UpBeat | Add Radio.SongProfile

### PR DESCRIPTION
**Pull request by marinofranz**

### CODE REVIEW

File: websites/U/UpBeatRadio/dist/metadata.json [hunk 1]
```diff
@@ -3,6 +3,12 @@
     "name": "Bas950",
     "id": "241278257335500811"
   },
+  "contributors": [
+    {
+      "name": "Marino",
+      "id": "598953747963707409"
+    }
+  ]
   "service": "UpBeatRadio",
```
Comment (ririxichan) [inline websites/U/UpBeatRadio/dist/metadata.json:12]: fix json.

**Review by ririxichan — CHANGES_REQUESTED** [review]

File: websites/U/UpBeatRadio/presence.ts [hunk 2]
```diff
@@ -103,6 +103,11 @@ presence.on("UpdateData", async () => {
       presenceData.details = "Viewing the";
       presenceData.state = "recently played songs";
       presenceData.smallImageKey = "reading";
+    } else if(document.location.pathname.includes("Radio.SongProfile")) {
+      const SongProfileArtist = document.querySelector("#mainContent > div > div:nth-child(1) > div > div.row.equal.p-md.d-flex > div.col-md-9 > div.infoContainer > div > div.artist").textContent;
+      const SongProfileTitle = document.querySelector("#mainContent > div > div:nth-child(1) > div > div.row.equal.p-md.d-flex > div.col-md-9 > div.infoContainer > div > div.song").textContent;
```
Comment (Bas950) [inline websites/U/UpBeatRadio/presence.ts:108]: merge consts

File: websites/U/UpBeatRadio/dist/metadata.json [hunk 3]
```diff
@@ -3,13 +3,19 @@
     "name": "Bas950",
     "id": "241278257335500811"
   },
+  "contributors": [
+    {
+      "name": "Marino",
+      "id": "598953747963707409"
+    }
+  ],
```
Comment (Bas950) [inline websites/U/UpBeatRadio/dist/metadata.json:11]: not enough has changed to be able to add yourself as contributor

**Review by Bas950 — CHANGES_REQUESTED** [review]
upload proof of it working

File: websites/U/UpBeatRadio/dist/metadata.json (hunk 3 above)
Comment (marinofranz) [reply websites/U/UpBeatRadio/dist/metadata.json:11]: ririxi said so, but ok

Comment (ririxichan) [reply websites/U/UpBeatRadio/dist/metadata.json]: oof my bad

**Comment by marinofranz** [comment]
image

**Review by Bas950 — APPROVED** [review]

**Review by Alanexei — APPROVED** [review]

**Pull request merged**